src/main/java/org/folio/invoices/utils/HelperUtils.java

```diff
@@ -164,7 +164,9 @@ public static String getEndpointWithQuery(String query, Logger logger) {
     return future;
   }
 
-
+  public static String buildQuery(String query, Logger logger) {
```

Comment:
Such method is already available but named a bit differently: `getEndpointWithQuery`

Reply:
Sure, I'm working on resolving merge conflicts and will be refactoring most of the flies.